packages/sdk/src/tests/nft.test.ts

```diff
@@ -0,0 +1,45 @@
+import { ApillonConfig } from '../lib/apillon';
+import { Nft } from '../modules/nft/nft';
+import { getConfig } from './helpers/helper';
+
+describe('Nft tests', () => {
+  let config: ApillonConfig;
+
+  beforeAll(async () => {
+    config = getConfig();
+  });
+
+  test.only('list nft collections', async () => {
+    const nft = new Nft(config);
+    try {
+      const collection = await nft.list();
+      console.log(collection);
+    } catch (e) {
+      console.log(e);
+    }
+  });
+
+  test('get nft collection details', async () => {
```

Comment:
This test gives you list of transactions, not collection details!

Reply:
Please don't review the code yet since its not complete. Only the overall arhitecture :)